src/tests/grid/test_grids.cc

```diff
@@ -466,8 +472,8 @@ CASE( "test_cubedsphere" ) {
             double tolerance         = 1e-13;
             // Expected latitudes/longitude per tile
 
-            if ( ( s == "CS-EA-" + std::to_string( resolution ) )  ||
-                 ( s == "CS-ED-" + std::to_string( resolution ) ) ) {
+            if ( ( s == "CS-EA-L-" + std::to_string( resolution ) )  ||
+                 ( s == "CS-ED-L-" + std::to_string( resolution ) ) ) {
```

Comment:
I still need to add `else if` blocks to check the results of the cell-center grids.

Reply:
Yes, you do.